scripts/install_obsolete_jsoncpp_1_7_4.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+set -e
+
+TEMPDIR=$(mktemp -d)
+(
+    cd $TEMPDIR
+    wget https://github.com/open-source-parsers/jsoncpp/archive/1.7.4.tar.gz
+    tar xvzf "1.7.4.tar.gz"
+    cd "jsoncpp-1.7.4"
+    mkdir -p build/debug
```

Comment:
`build` would be enough as a directory and that is our usual name we follow, though this isn't a blocker.

Reply:
addressed